### PR TITLE
refactor: apply final/abstract to various fawe extent classes

### PR DESCRIPTION
**Pull request by dordsor21**

### CODE REVIEW

**Comment by dordsor21** [comment]
Thoughts?

**Comment by dordsor21** [comment]
I'll fix this lol

File: worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/TransformExtent.java [hunk 1]
```diff
@@ -12,6 +12,12 @@
 import com.sk89q.worldedit.world.block.BlockState;
 import com.sk89q.worldedit.world.block.BlockStateHolder;
 
+/**
+ * Unused internal, will be removed
+ *
+ * @deprecated Will be removed - unused internally
+ */
+@Deprecated(forRemoval = true, since = "TODO")
```
Comment (SirYwell) [inline worldedit-core/src/main/java/com/fastasyncworldedit/core/extent/TransformExtent.java:20]: Would it make sense to remove it in v3 and deprecated it in v2 already?

**Comment by SirYwell** [comment]
You removed the comment from `BlockTransformExtent` but you didn't make it final, I assume that was unintentional?

**Comment by dordsor21** [comment]
> You removed the comment from `BlockTransformExtent` but you didn't make it final, I assume that was unintentional?

Yeah I realised it was a WE class

**Review by SirYwell — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
`final`ly

**Pull request merged**